cp-connector/pkg/nats/nats.go

```diff
@@ -4,6 +4,8 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	cloudevents "github.com/cloudevents/sdk-go/v2"
```

Comment:
i think we can completely avoid this dependency by creating just our own constant:
```
const CloudEventsVersionV1 = "1.0"
```

Reply:
agree, will replace that